### DISCUSSION TITLE
Piece with realtime generation of scores

### DISCUSSION THREAD

**Discussion by charlesneimog** [DigiScore/neoscore#98] [Show and tell]
Hi,

Sharing a piece with neoscore. The III mov of the piece is composed using FFT and Partial-Tracking. The FFTs are calculated in smartphones, and the idea is to show the scores in the Smartphones too. 

It needs, Python 3.11, Puredata.
In PureData it needs this externals: 
- cyclone (library), 
- else (library), 
- py4pd https://github.com/charlesneimog/py4pd, 
- server https://github.com/charlesneimog/pd-server. 

The repo is here: https://github.com/charlesneimog/ecos

**Comment by ajyoon** [comment]
Thanks for sharing! Any chance you could share a screenshot or video of it in action? I'd love to make a little gallery of real-life music using neoscore.

**Comment by charlesneimog** [comment]
The piece will be recorded (video and audio) soon, then I will be happy to share it.

**Comment by charlesneimog** [comment]
Hi, here is the video of the first version of the piece!

https://www.youtube.com/watch?v=XIEI7-W7t2o

It is not completely ready yet...

It is not possible to see what is happen, but I upload an example in my web site:

https://www.charlesneimog.com/pages/flute/flute.html

Every time the bar below reaches the end, the page sends a signal to puredata which generates new scores.

**Comment by craigvear** [comment]
Excellent work Charles. Hope to see it performed some day.
